man/tpm2_readpublic.1.md

```diff
@@ -22,9 +22,7 @@
 
     An optional file to save the name structure of the object.
 
-  * **-f**, **\--format**=_FORMAT_:
-
-    Public key format.
+[pubkey options](common/pubkey.md)
```

Comment:
@idesai not really, because the common/pubkey.md includes the `-f option.`

```md
  * **-f**, **\--format**:

    Format selection for the public key output file. 'tss' (the default) will
    output a binary blob according to the TPM 2.0 Specification. 'pem' will
    output an OpenSSL compatible PEM encoded public key. 'der' will output an
    OpenSSL compatible DER encoded public key. 'tpmt' will output a binary blob
    of the TPMT_PUBLIC struct referenced by TPM 2.0 specs.
```